openpype/lib/attribute_definitions.py

```diff
@@ -452,21 +457,34 @@ def __init__(self, key, items, default=None, **kwargs):
 
         self.items = items
         self._item_values = set(item_values)
```

Comment:
```suggestion
        self._item_values = item_values
```
Assuming the `item_values` is changed to set already above - see other review suggestion.